core/templates/dev/head/services/contextual/DeviceInfoService.ts

```diff
@@ -17,23 +17,37 @@
  */
 
 // See: https://stackoverflow.com/a/11381730
```

Comment:
Move this comment above line `export class DeviceInfoService`.

Reply:
Done